package.json

```diff
@@ -95,9 +95,11 @@
   },
   "dependencies": {
     "@emotion/css": "^11.1.3",
-    "@grafana/data": "9.2.1",
-    "@grafana/runtime": "9.2.1",
-    "@grafana/ui": "9.2.1",
+    "@grafana/data": "canary",
```

Comment:
Hmm I need to think about how to deal with this for now.  I think maybe we don't update the dependencies so we definitely aren't breaking anything, and if we need to do dev on scenes we modify the package.json each time